.github/workflows/release.yml

```
@@ -136,6 +136,25 @@ jobs:
+    security-analysis-with-zizmor:
+      name: Run zizmor
+      runs-on: ubuntu-latest
+      permissions:
+        security-events: write
+        contents: read
+        actions: read
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v6
+          with:
+              ref: ${{ needs.set-tags.outputs.git_ref }}
```

actionlint (1.7.9)

property "set-tags" is not defined in object type {}

(expression)